security_monkey/tests/watchers/test_iam_role.py

```diff
@@ -72,7 +74,7 @@ def pre_test_setup(self):
 
         for x in range(0, self.total_roles):
             # Create the IAM Role via Moto:
-            aspd["Statement"][0]["Resource"] = ARN_PREFIX + ":iam:012345678910:role/roleNumber{}".format(x)
+            aspd["Statement"][0]["Resource"] = ARN_PREFIX + "arn:aws:iam:012345678910:role/roleNumber{}".format(x)
```

Comment:
is `ARN_PREFIX` not set to anything?

Reply:
The test was fine, it passes for me locally, I could not figure out why it was failing in CI.  I believe this is something to do with module updates for python3 and how items resolve.  This will be specifically updated with the next update realigning the boto implementation.